.travis/travis.yml

```diff
@@ -0,0 +1,12 @@
+language: go
+go:
+    - 1.14
+
+script:
+    - go test -v
```

Comment:
@codepope my understanding was that this runs the test in Go, but that may not be the case here?

Forgive my ignorance, I've never written any Go and the repo here doesn't have instructions for running the tests yet, so I most likely made a bad assumption.